### DISCUSSION TITLE
Disable coordinate search in the forward geocoding

### DISCUSSION THREAD

**Discussion by galewis2** [osm-search/Nominatim#3017] [Q&A] [answered]
I have an app that uses Nominatim under the hood, and the feature that enables coordinate search in the forward geocoding gives some confusing results, especially as the results are the nearest addressable point, not the input coordinate on the map.

Is there a way to prevent responses for coordinates in the main search?

**Comment by mtmail** [comment, marked as answer]
It's the default behaviour and there's no configuration to switch it off. The logic is in `setNearPointFromQuery` in the file `lib-php/SearchContext.php` and gets called from `lib-php/Geocode.php` if you want to change it.

**Reply by galewis2** [reply]
SWEET. I looked at that and if I'm understanding what I'm seeing, it looks pretty easy to patch, or at least it's a great jumping off point. Thanks!
